monero-outreach-docs/en/quickfacts/quickfacts.md

```diff
@@ -51,42 +52,42 @@ Monero has made several large improvements since launch. Nearly all improvements
 
 The word Monero is from the Esperanto language. The creators chose to use Esperanto because it is a ‘decentralized’ language and represents the breaking of barriers between people, on a global scale. In Esperanto, Monero is a word composed of three elements freely put together, one syllabus each: mon + er + o. Each has a meaning.
 
-**mon:** money    
-**er:** the smallest part    
-**o:** a thing (grammatically speaking: a noun)   
+**mon:** money
+**er:** the smallest part
+**o:** a thing (grammatically speaking: a noun)
```

Comment:
This block will show up on only one line if the trailing spaces at the end are removed (4 were too many btw)

Reply:
Had to leave spaces between sentences because adding spaces at the end didn't work. Now I think this should be ready to merge.